docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -38,37 +43,144 @@ $ sudo usermod -a -G docker $(id -u -n)
 $ exit
 ```
 
-## Prepare network and UFS volume
+## Prepare Docker Volume to Presist the UFS Data 
+
+By default all files created inside a container are stored on a writable container layer. 
+The data doesn’t persist when that container no longer exists. Docker volumes are the preferred way
+to save data outside the conatiners. The following two types of Docker volumes are used the most:
+
++ **Host Volume**: You manage where in the Docker host's filesystem to store and share the 
+containers data. To create a host volume, run:
+  ```console
+  docker run -v /path/on/host:/path/in/container ...
+  ```
+  The file or directory is referenced by its full path on the Docker host. It can exist on the Docker host already, or it will be created automatically if it does not yet exist. 
+ 
+ 
++ **Named Volume**: Docker manage where they are located.  It should be be referred to by specific names. 
+To create a named volume, run:
+  ```console
+  docker volume create volumeName
+  docker run -v  volumeName:/path/in/container ...
+  ```
+  
+Either host volume or named volume can be used for Alluxio containers. For purpose of test or 
+POC (Proof of Concept), the host volume is recommanded, since it is the easiest type of volume 
+to use and very performant. More important, you know where to refer to the data in the host 
+filesystem and you can manipulate the files directly and easily outside the containers. 
+
+Therefore, we will use the host volume and mount the host directory ```/alluxio_ufs```  to the 
```

Comment:
Single backtick is fine for in-line highlighting, triple backtick is for multi-line